systemtests/src/test/java/io/enmasse/systemtest/sharedinfra/DefaultsTest.java

```diff
@@ -24,7 +25,7 @@
 
     @Test
     @DefaultMessagingInfrastructure
-    public void testDefaultInfra() {
+    public void testDefaultInfra(ExtensionContext extensionContext) {
```

Comment:
Im not sure if I understand it correctly, you mean create test context as a extension which can work with resource manager, and paas it into tests? This will require that every of our test must have displayName annotation, also this does not allow us to use default infra annotation I think.

Reply:
@kornys The tests are already taking this parameter ExtensionContext. I was just thinking to rename that, and interact with that instead of the resource manager.

Reply:
but extension context is junit5 stuff, it is not a custom class we created

Reply:
:(

Reply:
becasue you are sad @lulf  I will try to check if we can overide extension context and use custom object without overriding display name